structures/README.adoc

```diff
@@ -0,0 +1,54 @@
+= Automation structures
+include::../_style/render.adoc[]
+
+Before we start to describe roles, playbooks, etc, we need to decide which one we use for what.
+This section is meant for topics which are across multiple structures and don't fit nicely within one.
+
+.[big]#**Define which structure to use for which purpose**#
+[%collapsible]
+====
+Explanations::
+define for what you want to use roles, playbooks, potentially workflows (in Ansible Tower/AWX), and how to split the code you write.
+
+Rationale::
+especially when writing automation in a team, it is important to have a certain level of consistence and make sure everybody has the same understanding.
+By lack of doing so, your automation becomes unreadable and difficult to grasp for new members or even for existing members.
++
+This structure will also help you to have a consistent level of modelization so that re-usability becomes easier.
+Metaphorically speaking, only if stones have been cut at roughly the same size, can they be properly used to build a house.
```

Comment:
I added the sentence "If one team member uses roles where another one uses playbooks, they will both struggle to reuse the code of each other." does it make it clearer?

Reply:
I think what threw me off was "This structure", since "structures" has a different meaning in the the test so far.
Sorry, I don't have a good solution for this.

If you don't have one either we can resolve this.